language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/InterfaceTree.scala

```diff
@@ -123,7 +123,7 @@ private[codegen] final case class TypeWithContext(
 private[codegen] object InterfaceTree extends StrictLogging {
 
   def fromInterface(interface: Interface): InterfaceTree = {
-    val builder = InterfaceTreeBuilder.fromPackageId(interface.packageId)
+    val builder = new InterfaceTreeBuilder(new mutable.HashMap())
```

Comment:
I inlined the work done by a constructor here since the package ID was never really used and this is the only place where this was called.